gamecenter/public/views.py

```diff
@@ -5,12 +5,13 @@
 blueprint = Blueprint('public', __name__, static_folder="../static")
 
 
-@blueprint.route("/", methods=["GET"])
+@blueprint.route("/")
 def home():
-    # Handle logging in
+    """Return the home page"""
```

Comment:
Your pep8 linter is inconsistently yelling at you? You haven't done this to all functions. Not saying you have to, just odd.